templates/settings-admin.php

```diff
@@ -69,6 +67,27 @@ function FormatMegabytes(int $byte): string {
 				<p><?php p($l->t('Server time:')); ?> <strong id="numFilesStorage"><span class="info" id="servertime"></span></strong></p>
 				<p><?php p($l->t('Uptime:')); ?> <strong id="numFilesStorage"><span class="info" id="uptime"></span></strong></p>
 			</div>
+
+			<div class="col col-6 col-l-12">
+				<h2>
+					<img class="infoicon" src="<?php p(image_path('serverinfo', 'app-dark.svg')); ?>">
+					<?php p($l->t('Temperature')); ?>
+				</h2>
+				<div class="table-wrapper">
+					<table class="server-infos-table">
+						<thead>
+						</thead>
+						<tbody>
+						<?php foreach ($_['thermalzones'] as $thermalzone): ?>
+						<tr>
+							<td><?php p($thermalzone['type']) ?>:</td>
+							<td><span class="info" id="<?php p($thermalzone['type']) ?>_temp"></span>°C</td>
```

Comment:
Are we 100% sure that there can be no duplicate type?
Should we maybe use types as key in the array returned by getThermalZones to make that requirement clear?

Reply:
Duplicate type? Could you describe how it looks like?
There should be only one file `type` for each thermalzone, `/sys/class/thermal/thermal_zone*/temp`.

Reply:
```
$ cat /sys/class/thermal/thermal_zone*/{type,temp}
acpitz
acpitz
pch_haswell
INT3400 Thermal
x86_pkg_temp
52800
52800
49500
20000
52000
```

Indeed I see this on my laptop. Not sure if those 2 `acpitz` are the same thing, though.

Reply:
To update the data in the frontend properly on an xhr update we need some to identify the right sensor. Please add add a new property 'zone' to the $tzone array and set md5($thermal_zone) as value. The hashed path should be stable enough. 

And please change snake_case (e.g. $thermal_zones) to camelCase ($thermalZones).

Thanks!

Reply:
Thank you very much!

The hopefully last step is to change

`<td><span class="info" id="<?php p($thermalzone['type']) ?>_temp"></span>°C</td>`

to 

`<td><span class="info" id="<?php p($thermalzone['hash']) ?>"></span>°C</td>`

(to use hash as unique identifier for the html elment)

and update `script.js` to also use the hash

`document.getElementById(data.thermalzones[i]['type'] + "_temp").innerHTML = data.thermalzones[i]['temp'];`

to 

`document.getElementById(data.thermalzones[i]['hash']).innerHTML = data.thermalzones[i]['temp'];`

(note: I don't have thermal sensors in my /sys so I cannot test myself)

Reply:
Done. It still looks good for my sensors. Please do let me know if there is something can be improved.

Reply:
Thank you very much!